src/clr-angular/forms/combobox/_combobox.clarity.scss

```diff
@@ -1,69 +1,188 @@
+    @include form-fields-appearance($clr-forms-border-color);
+
+    // override the above mixin for combobox-specific styles
+    @include css-var(
+      background-color,
+      clr-ng-select-input-background-color,
+      $clr-ng-select-input-background-color,
+      $clr-use-custom-properties
+    );
+
+    border-bottom: $clr-global-borderwidth solid $clr-forms-border-color;
+
+    // the border color is on the component; we don't want it on the input itself
+    .clr-input.clr-combobox-input:focus {
+      background: none;
+    }
+
+    // ideally I want to just use &:focus-within, but cross-browser support is limited
+    // for now, we programmatically add/remove .clr-focus class to achieve this
+    &:focus-within,
+    &.clr-focus {
+      border-bottom: $clr_baselineRem_2px solid $clr-forms-focused-color;
+    }
+
+    &.invalid,
+    &.invalid:focus,
+    &.clr-focus.invalid {
+      border-bottom-color: $clr-forms-invalid-color;
+    }
 
     //Others
-    outline: none;
+    .clr-combobox-input {
+      background: none;
+      border-bottom: none;
+    }
+
+    .clr-combobox-remove-btn {
+      background: transparent;
+      border: none;
+      cursor: pointer;
+      padding: $clr_baselineRem_4px $clr_baselineRem_4px;
+      color: $clr-ng-select-pill-font-color;
+    }
+
+    .label-combobox-pill {
+      margin: $clr_baselineRem_0_125 $clr_baselineRem_0_125 0 0;
+      @include css-var(
+        background-color,
+        clr-ng-select-pill-background-color,
+        $clr-ng-select-pill-background-color,
+        $clr-use-custom-properties
+      );
+      border-width: $clr_baselineRem_1px;
+      border-style: solid;
+      @include css-var(
+        border-color,
+        clr-ng-select-pill-border-color,
+        $clr-ng-select-pill-border-color,
+        $clr-use-custom-properties
+      );
+
+      padding: 0 $clr_baselineRem_4px;
+
+      .clr-combobox-pill-content {
+        @include css-var(
+          color,
+          clr-ng-select-pill-font-color,
+          $clr-ng-select-pill-font-color,
+          $clr-use-custom-properties
+        );
+        font-size: $clr-typography-xxsmalltext;
+        font-weight: $clr-font-weight-regular;
+        padding: 0 $clr_baselineRem_8px 0 $clr_baselineRem_4px;
+      }
+    }
+
+    .clr-combobox-input-wrapper {
+      border: none;
+      background: none;
+    }
   }
 
   .clr-combobox-trigger {
     //Dimensions
-    @include equilateral($clr-ng-select-trigger-dimensions);
-    padding: 0;
-    margin: 0;
+    width: $clr-ng-select-trigger-dimensions;
+    margin: auto;
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    right: 0;
 
     //Other
     background: none;
     border: none;
     @include css-var(color, clr-ng-select-trigger-color, $clr-ng-select-trigger-color, $clr-use-custom-properties);
     cursor: pointer;
     outline: none;
-    transform: translateX(-1 * $clr-ng-select-trigger-dimensions);
   }
 
   .clr-options {
```

Comment:
Would like to make sure we prefix classes fully, so `.clr-combobox-options` and update this throughout the CSS.